src/pal/src/configure.cmake

```diff
@@ -366,6 +366,9 @@ check_cxx_source_runs("
 #include <stdlib.h>
 #include <time.h>
 #include <sys/time.h>
+#ifdef (__linux__)
```

Comment:
Please use `#ifdef __LINUX__` instead, that's what we use in the code base. Btw, the `#ifdef (__linux__)` has a syntax error, there should be just the symbol, no parentheses.